app/models/manageiq/providers/openstack/base_metrics_capture.rb

```diff
@@ -92,30 +92,34 @@ def list_meters(filter)
   def meter_names(metric_capture_module)
     @meter_names ||= metric_capture_module::COUNTER_NAMES
   end
+  
+  cache_with_timeout(:counters_by_vm, 60.minutes) { Hash.new }
```

Comment:
I just moved this out of the `included do; end` block and removed the `extend ActiveSupport::Concern` because this is a class not a module on ivanchuk